pyopae/test_pyopae.py

```diff
@@ -37,84 +37,154 @@
 MOCK_PORT_ERROR = "/tmp/class/fpga/intel-fpga-dev.0/intel-fpga-port.0/errors/errors"
 
 class TestProperties(unittest.TestCase):
+    def test_set_parent(self):
+        props = opae.fpga.properties()
+        toks = opae.fpga.enumerate([props])
+        props2 = opae.fpga.properties(type=opae.fpga.ACCELERATOR,
+                                     parent=toks[0])
+        assert props2.parent
+        props2 = opae.fpga.properties(type=opae.fpga.ACCELERATOR)
+        props2.parent = toks[0]
+        assert props2.parent
+
     def test_guid(self):
         props = opae.fpga.properties(guid=NLB0)
         guid_str = props.guid
         guid = uuid.UUID(guid_str)
         assert str(guid).lower() == NLB0
+        props = opae.fpga.properties()
+        props.guid = NLB0
+        guid_str = props.guid
+        guid = uuid.UUID(guid_str)
+        assert str(guid).lower() == NLB0
 
     def test_set_objtype_accelerator(self):
         props = opae.fpga.properties(type=opae.fpga.ACCELERATOR)
         assert props.type == opae.fpga.ACCELERATOR
+        props = opae.fpga.properties()
+        props.type = opae.fpga.ACCELERATOR
+        assert props.type == opae.fpga.ACCELERATOR
 
     def test_set_objtype_device(self):
         props = opae.fpga.properties(type=opae.fpga.DEVICE)
         assert props.type == opae.fpga.DEVICE
+        props.type = opae.fpga.DEVICE
```

Comment:
We've already asserted that the type is of DEVICE on line 70 so we should probably set the type to something different than what it was before.